cluster/Vagrantfile

```diff
@@ -11,6 +11,7 @@ UBUNTU = 'ubuntu'.freeze
 CENTOS = 'centos'.freeze
 RHEL = 'rhel7'.freeze
 
+BOX_VERSION= ENV['BOX_VERSION'] || '1707.01'
```

Comment:
not a blocker and I am not a ruby guy, but the other assignments have a space before '='

Reply:
Haha, yeah there should be a space here for consistency reasons even though this is still valid Ruby.  Maybe also throw a `.freeze` on the end of the string literal too